src/integration-tests/introspector/introspectTest.sh

```diff
@@ -393,6 +420,48 @@ function deployCreateDomainJobPod() {
   waitForPod $pod_name
 }
 
+#
+# Create the model in image docker image
+#
+function createMII_Image() {
+  trace "Info: Create MII Image"
+
+  mkdir -p ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cp ${SCRIPTPATH}/mii/models/*  ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cd ${SCRIPTPATH}/mii/workdir || exit 1
+  export WORKDIR=${SCRIPTPATH}/mii/workdir || exit 1
+  export MODEL_IMAGE_TAG=it || exit 1
+  export MODEL_IMAGE_NAME=model-in-image || exit 1
+
```

Comment:
When setting up env vars that are very specific to external scripts, consider encasing the entire set of calls into a "(" ... script calls ... ")".  This will make it clear (self documenting) they're scoped only for the external script and will make sure they don't accidentally leak to other calls to the external script in the global scope.

Reply:
done